subjects/cybersecurity/osint-master/README.md

```diff
@@ -0,0 +1,164 @@
+## OSINT-Master
+
+<center>
+<img src="./resources/osint-meme.png?raw=true" style = "width: 673px !important; height: 439px !important;"/>
+</center>
+
+### Introduction:
+Open-source intelligence (OSINT) is a key component of cybersecurity, providing valuable insights into potential vulnerabilities and security risks. This project involves creating a tool that performs comprehensive passive reconnaissance using publicly available data.
+
+### Objective:
+The goal is to build a multi-functional using a programming language of your choice, The tool is capable of retrieving detailed information based on user inputs such as `Full names`, `IP addresses`, `usernames`, and `domains`. This project will enhance your skills in data analysis, ethical considerations, and the use of various cybersecurity tools and APIs.
+
+By completing this project, You will:
+- Develop an understanding of OSINT techniques and their applications.
+- Gain practical experience in programming, API integration, and data handling.
+- Learn to identify and mitigate security risks, including subdomain takeovers.
+- Understand the ethical and legal implications of cybersecurity practices.
+
+### Resources
+Some useful resources:
+- [Open-source intelligence](https://en.wikipedia.org/wiki/Open-source_intelligence)
+- [Doxing](https://en.wikipedia.org/wiki/Doxing)
+- [Kali Tools - Recon](https://en.kali.tools/all/?category=recon)
+- [OSINT Tools on GitHub](https://github.com/topics/osint-tools)
+
+Before asking for help, ask yourself if you have really thought about all the possibilities.
+
+### Role play
+To enhance the learning experience and assess your knowledge, a role play question session will be included as part of this project.
+This section will involve answering a series of questions in a simulated real-world scenario where you assume the role of a Cyber Security Expert explaining how to protect information from OSINT techniques to a team or stakeholder.
+
+The goal of the role play question session is to:
+- Assess your understanding of OSINT risks and mitigation strategies.
+- Test your ability to communicate effectively and explain security measures related to this project.
+- Challenge you to think critically about the importance of information security and consider alternative approaches.
+- Explain what subdomain takeovers are.
+
+Prepare for a role play question session in the audit.
+
+### Project Requirements
+#### Input Handling:
+The tool should accept the following inputs: `Full Name`, `IP Address`, `Username`, and `Domain`.
+
+#### Information Retrieval:
+- Full Name:
+Parse the input to extract "First Name" and "Last Name".
+Look up associated information such as phone numbers, addresses, and social media profiles using directory APIs or web scraping.
```

Comment:
If I were a learner starting this project, I would like to have some extra information about `... using directory APIs or web scraping.`
For example:
- Is there a minimum of resources I should check to retrieve information from? 
- Is there a specific set of information that I should try to retrieve? For example, it should be at least the one listed in the example or fewer details is enough?

Reply:
I added this line to the subject:
> You are responsible for choosing the way you want to find the data and for using the API's. be aware of the API's `Terms of Use` and `Cost` before use!

Reply:
The added information does not reply to the 2 questions mentioned.